app/components/homeScreen/BabyNotification.tsx

```diff
@@ -54,7 +54,9 @@ const BabyNotification = ():any => {
                         source={{ uri: 'file://' + CHILDREN_PATH + activeChild.photoUri }}
                       ></ImageIcon>
                     ) : (
-                      <Icon name="ic_baby" size={36} color="#000" />
+                      activeChild.gender != '' ? activeChild.gender == 'Boy'?<Icon name="ic_baby" size={36} color="#000" />:
+                      <Icon name="ic_baby" size={36} color="#000" />:
+                     <Icon name="ic_baby_girl" size={36} color="#000" />

```

Comment:
In both cases it displays the same icon. Please correct the code. I assume it should be as

```
{activeChild.gender !== '' ? 
  (activeChild.gender === 'Boy' ? 
    <Icon name="ic_baby_boy" size={36} color="#000" /> :
    <Icon name="ic_baby_girl" size={36} color="#000" />) :
  <Icon name="ic_baby_girl" size={36} color="#000" />
```